charts/zwave2mqtt/values.yaml

```diff
@@ -1,105 +1,57 @@
 # Default values for zwave2mqtt.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-# upgrade strategy type (e.g. Recreate or RollingUpdate)
-strategyType: Recreate
 
 image:
   repository: robertslando/zwave2mqtt
-  tag: 4.0.3
   pullPolicy: IfNotPresent
+  tag: 4.0.6
 
-# timezone to run the service as
-# timezone: "America/New York"
-
-# device where the zwave device can be accessed
-device: "/dev/ttyASM0"
-
-imagePullSecrets: []
-nameOverride: ""
-fullnameOverride: ""
+strategy:
+  type: Recreate
 
-# Probes configuration
-probes:
-  liveness:
-    initialDelaySeconds: 30
-    failureThreshold: 5
-    timeoutSeconds: 10
-  readiness:
-    initialDelaySeconds: 30
-    failureThreshold: 5
-    timeoutSeconds: 10
-
-persistence:
-  enabled: false
-  ## zwave2mqtt configuration data Persistent Volume Storage Class
-  ## If defined, storageClassName: <storageClass>
-  ## If set to "-", storageClassName: "", which disables dynamic provisioning
-  ## If undefined (the default) or set to null, no storageClassName spec is
-  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
-  ##   GKE, AWS & OpenStack)
-  ##
-  # storageClass: "-"
-  ##
-  ## If you want to reuse an existing claim, you can pass the name of the PVC using
-  ## the existingClaim variable
-  # existingClaim: your-claim
-  accessMode: ReadWriteOnce
-  size: 1Gi
-  ## Do not delete the pvc upon helm uninstall
-  skipuninstall: false
+env:
+  # TZ
```

Comment:
Maybe leave this out, since a default (example) is already set in `common`?

Reply:
There's a default environment variable that needs to be set. or are you just talking about the TZ line?

Reply:
Yes, I was only talking about the `# TZ` line 🙂